tx_submission_test.go

```diff
@@ -48,7 +48,7 @@ func testSubmitTxResult(t *testing.T) filepath.WalkFunc {
 	return func(path string, info fs.FileInfo, err error) error {
 		t.Run(filepath.Base(path), func(t *testing.T) {
 			if err != nil {
-				t.Fatalf("got %v; want nil", err)
+				t.Fatalf("Ogmios files missing. You may need to `rm -rf ext/ogmios/ && git clone git@github.com:CardanoSolutions/ogmios.git ext/ogmios/")
```

Comment:
Is it possible to warn people instead of causing everything to fail?

Reply:
This is fine, as long as it gives resolution instructions

Reply:
I think it should fail personally.

Reply:
I think I'm just uncomfortable with tests that require external data being run as plain old unit tests. That and the instructions could break depending on how the tests are structured in the future. (They've bounced around a couple of times.) I guess it's not an integration test or a system test, though. Hmmm....